lib/azure/armrest/resource_provider_service.rb

```diff
@@ -53,7 +54,8 @@ def list
       def get(namespace)
         url = build_url(namespace)
         response = rest_get(url)
-        JSON.parse(response.body)
+        #JSON.parse(response.body)
+        Azure::Armrest::ResourceProvider.new(response.body)
```

Comment:
What's the proper way to convert an array of wrapper objects, for all `list` methods?